upi/azure/arm/deploy_azarm.sh

```diff
@@ -0,0 +1,13 @@
+echo "Start Deployment"
+export arm_template_path=`./gitpath.sh upi/azure/arm/azuredeploy.json`
+az group deployment create \
+   --name $1 \
+   --resource-group $1 \
+   --template-uri ${arm_template_path} \
```

Comment:
why don't we use `--template-file` and a local file instead of pulling from github?

Reply:
There is internal references for master and node templates, while could use template-file, the subtemplates still have to be pulled from git. So better all are pulled from git to be consistent.

Reply:
> the subtemplates still have to be pulled from git.

do you mind providing a little more context here. 

the official openshift docs most likely can't have a step of hey pull this content from this random url, It's best if every thing is based on local files.

Reply:
The main template has two subtemplates. They are pulled from the main template. If you use a file, you can only reference the main template. You cannot reference subtemplates. So if you pull the git repo, and then change the main template and the sub templates, the original subtemplates would be used. The url is the installer repo, github.com/openshift/installer, the extra  pathing you see is for it to pull the raw file from the installer repo. 

Fixed the path as well.

Reply:
Note this file has changed.

Reply:
There are 3 templates. the main azuredeploy.json, and master.json, and node.json, so if you use "file" it will only apply to azuredeploy.json, it better to have all 3 come from git.